oviewer/oviewer.go

```diff
@@ -360,10 +360,15 @@ const QuitSmallCountDown = 10
 // when the debug flag is enabled.
 const MaxWriteLog int = 10
 
+// The name of the converter that can be specified.
 const (
-	esConv    = "es"
-	rawConv   = "raw"
-	alignConv = "align"
+	esConv    string = "es"    // esConv processes escape sequence(default).
+	rawConv   string = "raw"   // rawConv is displayed without processing escape sequences as they are.
+	alignConv string = "align" // alignConv is aligned in each column.
+)
+
+const (
+	generalName string = "general"
```

Comment:
Usually constants are sharing a common prefix, allowing to find the easily when using completion in the IDE, they are also ordered in the godoc if they are exported

```suggestion
	convEscaped    string = "es"    // convEscaped processes escape sequence(default).
	convRaw   string = "raw"   // convRaw is displayed without processing escape sequences as they are.
	convAlign string = "align" // convAlign is aligned in each column.
)

const (
	nameGeneral string = "general"
```

Reply:
Thank you for your comment.
That's right. Change.